### PR TITLE
[lscache] Update version to v.1.3.0 

### PR DESCRIPTION
**Pull request by c-py**
Please fill in this template.

- [ X] Use a meaningful title for the pull request. Include the name of the package modified.
- [ X] Test the change in your own code. (Compile and run.)
- [ X] Add or edit tests to reflect the change. (Run with `npm test`.)
- [X ] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [X ] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [ X] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

If changing an existing definition:
- [ X] Provide a URL to documentation or source code which provides context for the suggested changes: https://github.com/pamelafox/lscache/blob/master/lscache.js
- [ X] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.

### CODE REVIEW

File: types/lscache/index.d.ts [hunk 1]
```diff
@@ -1,6 +1,7 @@
-// Type definitions for lscache v1.0.7
+// Type definitions for lscache v1.3.0
```
Comment (sandersn) [inline types/lscache/index.d.ts]: Definitely Typed doesn't retain patch versions, so you should just put `1.3` here.

File: types/lscache/index.d.ts [hunk 2]
```diff
@@ -1,6 +1,7 @@
...
 // Project: https://github.com/pamelafox/lscache
 // Definitions by: Chris Martinez <https://github.com/Chris-Martinezz>
+//               : c-py <https://github.com/c-py>
```
Comment (sandersn) [inline types/lscache/index.d.ts]: I don't think this is the accepted format for this field. Check something like react with lots of authors for an example.

**Review by sandersn — CHANGES_REQUESTED** [review]
It looks like the dt-header and npm-naming lint rules are disabled in tslint.json. Since you're changing the header, can you please re-enable at least dt-header? Preferably both unless npm-naming fails -- but it shouldn't, because lscache 1.3.0 is on npm.

**Comment by c-py** [comment]
Thanks @sandersn ! I have updated the PR with the suggested changes.

**Review by sandersn — APPROVED** [review]

**Pull request merged**